tests/console/apache.pm

```diff
@@ -102,7 +102,7 @@ sub run {
     # We stop current webserver and prepare another http2-prefork environment
     systemctl 'stop apache2';
 
-    if (is_sle('12-SP2+')) {
+    if (is_sle('12-SP2+') && !check_var('SYSTEM_ROLE', 'kvm')) {
```

Comment:
I would add a record info mentioning that the http2-prefork tests are being skipped, 'cause we're basically disabling half of the test. Or even better, let's not disable it by adding/using the missing files to the data directory

Reply:
We can't do that as apache get's installed from a different test module.
Also disabling this part of the test doesn't really decrease coverage as it still runs in the non-kvm scenario.

Reply:
Also I don't want to mess with the system by changing some configuration.
That would IMHO decrease the overall coverage as we wouldn't be testing anymore what a customer would experience in the kvm role.

Reply:
But I'll add the `record_info`